### ISSUE TITLE
Is pg-aws_rds_iam used by anyone in production?

### ISSUE THREAD

**Issue by brogersop** [haines/pg-aws_rds_iam#248]
We're looking at using this, but curious how many others use it in a production-like setting. 

We see ~ 16k downloads, how much is known about where this is being used?

Do any of the maintainers specifically use it for your own work?

**Comment by haines** [comment]
Hi! Yes, I'm using this at [work](https://www.zencargo.com/). We've had it running in two Rails applications in our pre-production environment and one in production for about a month now, and are promoting it to production for the other app next week.

We haven't had any issues with the gem itself, although we did run into a bug on AWS's end - if your database instance is too small for your workload, then IAM authentication can fail (we saw this as 503 responses from the token verification service in the PostgreSQL error logs).

AWS said they're aware of the issue and a fix is in progress, but in the meantime it's worth double-checking your instance type is large enough (we were using a `t3.micro` which was way too small for our load, the freeable memory was next to nothing).

Since we scaled up the database instance to something more sensible, we've had no issues.

I'm not sure if anyone else is running it in production; one person reached out to me via email a while back because they were having difficulty getting the default token generator to pick up their credentials, but otherwise this is the first issue opened on the repo... so either it's working perfectly or no-one is using it 😅

If you do give it a try, let me know how you get on!
